entity_mongo_model_opts.go

```diff
@@ -0,0 +1,31 @@
+package mgod
+
+import (
+	"github.com/Lyearn/mgod/schema/schemaopt"
+)
+
+type entityMongoModelOptions struct {
+	connOpts   connectionOptions
+	schemaOpts *schemaopt.SchemaOptions
+}
+
+type connectionOptions struct {
+	db   string
+	coll string
+}
+
+// NewEntityMongoModelOptions creates a new entityMongoModelOptions instance.
+// Its instance is used to provide necessary configuration options to the NewEntityMongoModel function.
+//
+// dbName is the name of the database in which the entity is stored.
+// collection is the name of the mongo collection in which the entity is stored.
+// schemaOpts is the schema level options for the entity.
+func NewEntityMongoModelOptions(dbName string, collection string, schemaOpts *schemaopt.SchemaOptions) *entityMongoModelOptions {
+	return &entityMongoModelOptions{
+		connOpts: connectionOptions{
```

Comment:
no. is it required though? because there are only 3 params to it - `db`, `collection` and `schemaOpts` where first two are straight forward and we do have documentation for the last one.

Reply:
can skip for now